site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -100,3 +100,9 @@ CQ - Active: True Reason: Ready Message: Can admit new workloads
 AC - Active: True Reason: Active Message: The admission check is active
 MC - Active: True Reason: Active Message: Connected
 ```
+
+## Setup MultiKueue with Open Cluster Management
+
```

Comment:
Add a line indicating that this is an option, not a compulsory step.

Reply:
Yes, updated the title and description with optional.